src/main.zig

```diff
@@ -190,32 +205,37 @@ const Context = struct {
     entry: Entry,
     instance: Instance,
     device: Device,
+    physical_devices: []c.VkPhysicalDevice,
```

Comment:
listed physical devices may change, so am not sure if it would be valid to cache it.